tests/integration/test_install.py

```diff
@@ -131,6 +133,45 @@ def check_valid_version(version: str):
             assert False, f"{version} does not look like a valid semver"
 
 
+class _WorkspaceInstaller(WorkspaceInstaller):
```

Comment:
Mock Installer if we are intending to run unit tests, if we are running proper integration tests this needs to go inside accpetance.yml which loads the credentials to run a proper install. 

@ericvergnaud  and @asnare